misk/src/main/kotlin/misk/web/BoundAction.kt

```diff
@@ -254,7 +254,14 @@ private class RequestBridgeInterceptor(
     if (returnValue is Response<*>) {
       httpCall.statusCode = returnValue.statusCode
       httpCall.addResponseHeaders(returnValue.headers)
+      val trailers = returnValue.trailers
```

Comment:
Defer resolving this until the response body has been streamed?

Reply:
I thought that the trailers are actually accumulated in [JettyUpstreamResponse](https://github.com/cashapp/misk/blob/master/misk/src/main/kotlin/misk/web/jetty/JettyServletUpstreamResponse.kt#L40) and conveyed in a lambda already such that it takes care of delaying the write until after the body is streamed?